includes/File_Type.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Class FileTypes
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories;
+
+/**
+ * Class FileTypes
+ *
+ * @package Google\Web_Stories
+ */
+class File_Type extends Service_Base {
```

Comment:
`File_Type` is very generic and doesn't convey what exactly this class does.

We have the `SVG` class to support uploading SVG files, so perhaps this should be renamed to `Video_Captions` to make it clear that it's about supporting uploading video captions.

-----

Aside: we could establish a new `Media` namespace for this class, `SVG`, and the existing `Media` class

Reply:
So it should be `VTT`? 

> Aside: we could establish a new Media namespace for this class, SVG, and the existing Media class

Happy to do this. Should I do this in this PR?

Reply:
I'd rather name it `Video_Captions` instead of `VTT`, I think. Nut not sure.

> Happy to do this. Should I do this in this PR?

A new PR would be good. Of course only if you think it makes sense :-) It's just something that came to mind.

Reply:
Done in this PR.